pims/bioformats.py

```diff
@@ -51,10 +51,11 @@ def _find_jar():
             return os.path.join(loc, 'loci_tools.jar')
 
     warn('loci_tools.jar not found, downloading')
-    return _download_jar()
+    return download_jar()
 
 
-def _download_jar(version='5.7.0'):
+def download_jar(version='5.8.2'):
```

Comment:
Suggest bumping to bioformats 6.1.0, given the latest release announcement today: https://www.openmicroscopy.org/2019/05/20/bio-formats-6-1-0.html